tests/unit/readers/cubeconfig/v1_0/test_cube_from_config_json.py

```diff
@@ -69,7 +69,7 @@ def test_build():
             output_directory=output,
             csv_path=csv,
             fail_when_validation_error_occurs=True,
-            validation_errors_file_out=Path(output, "validation_errors.json"),
+            validation_errors_file_name=Path(output, "validation_errors.json"),
```

Comment:
Again, shouldn't this just be the file name now? There are other lines like this in the same file.